src/ontology/mondo-edit.obo

```diff
@@ -279750,6 +279750,7 @@ synonym: "mitochondrial DNA depletion syndrome 2 (myopathic type)" RELATED [MOND
 synonym: "mitochondrial DNA depletion syndrome type 2" EXACT [MONDORULE:1]
 synonym: "mtDNA depletion syndrome, myopathic form" EXACT [Orphanet:254875]
 synonym: "MTDPS2" RELATED ABBREVIATION [MONDO:Lexical]
+synonym: "thymidine kinase 2 deficiency" EXACT [https://orcid.org/0000-0001-9310-0163, https://rarediseases.org/rare-diseases/thymidine-kinase-2-deficiency/]
```

Comment:
@nicolevasilevsky I think "thymidine kinase 2 deficiency" should be a broad synonyms, since it is possible that a deficiency in this thymidine kinase 2 lead to a disease that is different from a myopathic form of mitochondrial DNA depletion syndrome.

Reply:
I updated this